Moved color scheme toggle

JianLoong opened the pull request:
### Prerequisites

Put an `x` into the box(es) that apply:

- [x]  This pull request fixes a bug.
- [ ] This pull request adds a feature.
- [ ] This pull request introduces breaking change.

### Description

Describe what this pull request achieves.

Moved the position of the color scheme toggle to the bottom left of the screen.

### Issues Resolved

![hugo-coder-normal](https://user-images.githubusercontent.com/933858/95619138-dbee7f00-0ab9-11eb-940b-ba4e5e06df8b.png)

![hugo-coder-responsive](https://user-images.githubusercontent.com/933858/95619141-ddb84280-0ab9-11eb-9aad-ed4a55d2aaf7.png)


### Checklist

Put an `x` into the box(es) that apply:

#### General

- [x] Describe what changes are being made
- [x] Explain why and how the changes were necessary and implemented respectively
- [x] Reference issue with #418 if applicable

#### Resources

- [x] If you have changed any SCSS code, run `make release` to regenerate all CSS files



#### Contributors

- [x] Add yourself to `CONTRIBUTORS.md` if you aren't on it already

luizdepra commented on assets/scss/_base.scss:
```diff
@@ -217,3 +217,13 @@ img.emoji {
+  position: fixed;
+  width:3em;
+  height:3em;
+  font-size: 1.6em;
```
You can remove both `width` and `height` rules here. And set `bottom` and `left` to `2rem`.

luizdepra commented on layouts/partials/footer.html:
```diff
@@ -22,5 +22,10 @@
       {{ end }}
     </section>
+    {{ if not .Site.Params.hidecolorschemetoggle }}
+    <span id="dark-mode-toggle" class="">
```
I'm missing the on-hover effect here. Maybe changing from `span` to `a` should be enough. Also, please remove these empty `class` attribute.

luizdepra commented:
I think we should "steal" [LoveIt](https://themes.gohugo.io//theme/LoveIt/theme-documentation-basics/)'s Style for these buttons and put a rounded background under the icon.

Tell me if you have time to make these changes. If not, I can merge your changes in a separated branch and do all the fixes.

JianLoong commented:
I fixed this issue.

However, previously because the colour scheme toggle was on the navigation bar, thus SCSS is under the navigation category. So for now, I just changed it there. 

Visually however, the toggle seems to be under footer section partial but, the HTML for it is **still** under the **header.html** partial. 

Do you want it to moved into the **footer.html** partial instead? Which approach is more logical? Seems to me, that the question "Is colour toggle considered a navigation element?" arises.

For now, I opted to leave the footer partial as it is and made minor changes to the head.html and changed SCSS related to the color toggle in the navigation SCSS.

Feel free to merge the branch and obtain "ideas" from the other style;  :smirk:

luizdepra commented:
I think the toggle button should be neither in the header nor in the footer. It should be in a proper div inside the body. But, don't worry about that, I can move it later.

I'm going to merge this PR into a new branch this weekend and work on it. Thank you in advance. :)
